src/gui/tray/Window.qml

```diff
@@ -45,6 +45,13 @@ Window {
         userLineInstantiator.active = true;
     }
 
+    function openFileActivityDialog(displayPath, absolutePath) {

```

Comment:
at top or bottom but not in the middle to make it easier to read (don't mix different types of children
I would go with

- new property declarations
- new signal declarations
- new function declarations
- new item children declarations
- state machine declaration